internal/helper/tls.go

```diff
@@ -0,0 +1,89 @@
+package helper
```

Comment:
add copyright and can you move this under the client package and rename the package to proxy

Reply:
Updated @shreyassreenivas